payloads/external/edk2/Kconfig.dasharo

```
@@ -392,4 +392,11 @@ config EDK2_ENABLE_QUIET_BOOT_FEATURE
 	help
 	  Enables quiet boot option in the TianoCore Payload.
 
+config EDK2_GRAPHICAL_UPDATE_DISPLAY
```

Maybe name it like this to make it easier to understand out of context (better to have consistent naming in EDK and coreboot):
```suggestion
config EDK2_GRAPHICAL_CAPSULE_PROGRESS
```

---

thanks, i can never get the names right: https://github.com/Dasharo/coreboot/pull/696/commits/9242f1a7e05d48c442436fd43c92553eba413932